src/Logikfabrik.Umbraco.Jet/Extensions/PreValueCollectionExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Logikfabrik.Umbraco.Jet.Extensions
```

Comment:
That's a good point. The only reason I moved it there was to be able to create a unit test.

This is now moved to `DataTypeSynchronizer` as a private method